### ISSUE TITLE
[Bug report] When unequiping an item that changes the LP / SP, the bar is not always refreshed

### ISSUE THREAD

**Issue by Talmit** [Pyrdacor/Ambermoon.net#367]
**Describe the bug**
When unequiping an item that changes the LP / SP, the bar is not always refreshed. This does not seem to happen when the SP change to maximum / from maximum.

**To Reproduce**
1. Select a character that has already lost SP
2. Unequip an item that provides SP (e.g. Lich Crown)
3. Observe the SP indicator during equip / unequip

**Expected behavior**

**Screenshots**
Nelvin (with lost SP)
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/2a624f4f-ac67-45dd-8c43-fe3ceb00a956)
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/341dc994-3773-4f9c-80cd-a64b2ef26a38)

Sabine (with maximum SP); This works correctly
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/1e1e5c89-accc-430e-ac47-c97209696cba)
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/97924bef-d1bc-4ade-a3f7-2d40e061781c)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 1.9.3 / Ambermoon Advanced 1.3

**Additional context**
The first provided save has Nelvin equipped with the crown.

**Comment by Pyrdacor** [comment]
I don't think this is a bug. The item just influences the maximum, not the current value. The current value is only changed if it would exceed the maximum.

**Comment by Pyrdacor** [comment]
If it would influence the current value you could just unequip and re-equip the crown at 0 SP to gain SP. This could be exploited.

**Comment by Talmit** [comment]
It shouldn't influence the current values, but that's not what I was trying to describe.

If you take a look at the screenshots with Nelvin, in the first screenshot the max SP are at 223, while the current SP are at 118. This means he has ~53% of his SP available.
In the second screenshot he has max SP 198 and current SP 118. This makes for 59.5% of his SP available.
The bar next to Nelvin's portrait which indicates his SP stays the same with the item equipped as well as with the item uneqipped.

I noticed the same behaviour in another instance as well, namely after unequipping the "Mond Amulett" from my own character:
Initial (full HP / SP with Amulett):
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/862ffc27-32c9-4809-b573-601d862cdecd)
After unequipping the Amulett (full HP / SP without Amulett):
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/9dcceeaa-d84f-412a-bad4-c5c1a472fc3e)
After equipping the Amulett (partial HP / SP with Amulett):
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/7dd40f07-9e6b-4ef8-93b2-7fbdde62db4a)
After unequipping the Amulett (full HP / SP without Amulett, but the indicator shows missing HP next to the portrait):
![image](https://github.com/Pyrdacor/Ambermoon.net/assets/88923192/e0274709-d378-46c4-8732-1dd05d99719b)

If you take a look at the HP / SP and Max. HP / Max. SP you'll notice that in the first screenshot Aret has full HP / full SP.  I just unequipped the Mond Amulett. However, the blue / green bars indicating HP / SP do not refresh themselfes.
